src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ServerKeyVaultKeyTests.cs

```diff
@@ -26,22 +26,22 @@ public ServerKeyVaultKeyTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Fact(Skip = "TODO: only works for live mode. Mihymel will fix the test issue for Create-ServerKeyVaultKeyTestEnvironment")] 
```

Comment:
@payiAzure can we open an issue with the following tests outlined and assign it to Mihymel?

Reply:
@cormacpayne Open an issue for this test and assigned it to mihymel.
@mihymel FYI. Thanks
https://github.com/Azure/azure-powershell/issues/6211